docs/integrations/sources/shopify-migrations.md

```diff
@@ -1,5 +1,26 @@
 # Shopify Migration Guide
 
+## Upgrading to 2.1.0
+This version implements `Shopify GraphQL BULK Operations` to speed up the following streams:
+ - `Products`
+ - `Product Images`
+ - `Product Variants`
+
+* In the `Products` stream, the `published_scope` property is no longer available.
+* In the `Products` stream, the `images` property now contains only the `id` of the image. Refer to the `Product Images` stream instead.
+* In the `Products` stream, the `variants` property now contains only the `id` of the variant. Refer to the `Product Variants` stream instead.
+* In the `Products` stream, the `position` property is no longer available.
+* The `Product Variants` stream now has the cursor field `updated_at` instead of `id`.
+* In the `Product Variants` stream, the date-time fields, such as `created_at` and `updated_at`, now use `UTC` format without a timezone component.
+* In the `Product Variants` stream, the `presentment_prices.compare_at_price` property has changed from a `number` to an `object of strings`. This field was not populated in the `REST API` stream version, but it is correctly covered in the GraphQL stream version.
+* The `Product Variants` stream's `inventory_policy` and `inventory_management` properties now contain `uppercase string` values, instead of `lowercase`.
+* In the `Product Images` stream, the date-time fields, such as `created_at` and `updated_at`, now use `UTC` format without a timezone component.
+* In the `Product Images` stream, the `variant_ids` property is no longer available. Refer to the `Product variants` stream instead.
```

Comment:
I think `position` is also missing based on the output you provided, right?

Reply:
It's not present in the record for the TARGET version, but schema didn't change. I'll add this mention.

Reply:
Corrected.

Reply:
Hello, position is not missing but it always returns null values. Could you please investigate?